tasks/release/ChangelogController.php

```diff
@@ -38,8 +38,9 @@ function get($version_name = null) {
     if (!$changelog) {
       $changelog = self::FALLBACK_RECORD;
     }
-    $notes = $this->renderList($issues, Jira::RELEASENOTE_FIELD_ID);
-    return [$heading, $changelog, $notes];
+    $notes = $this->renderList($issues, JiraController::RELEASENOTE_FIELD_ID);
+    $release_id = $version['id'];
+    return [$heading, $changelog, $notes, $release_id];
```

Comment:
Do we need to add $release_id into this output? I can't see it used anywhere.

Reply:
Good catch, I originally used output of this method until I realized I need to fetch version info directly from JiraController anyway. So I guess it's forgotten here. Will fix.